packages/devtools_app/lib/src/screens/memory/panes/chart/chart_pane.dart

```diff
@@ -197,8 +197,6 @@ class _MemoryChartPaneState extends State<MemoryChartPane>
     return ValueListenableBuilder<bool>(
       valueListenable: preferences.memory.showChart,
       builder: (_, showChart, __) {
-        // TODO(polina-c): animate the showing / hiding of this chart.
-
```

Comment:
please add this TODO back (if you merge with master, this todo has actually been updated to include a link to a github issue)